modules/monitoring-enabling-the-platform-alertmanager-instance-for-user-defined-alert-routing.adoc

```diff
@@ -33,8 +34,10 @@ metadata:
   namespace: openshift-monitoring
 data:
   config.yaml: |
+    # ...
     alertmanagerMain:
-      enableUserAlertmanagerConfig: true <1>
+      enableUserAlertmanagerConfig: true #<1>
```

Comment:
Suggesting per [this](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-asciidoc-callouts-in-the-yaml) OCP guideline
```suggestion
      enableUserAlertmanagerConfig: true # <1>
```